docs/getting-started-guides/ubuntu/upgrades.md

```diff
@@ -49,14 +46,13 @@ After the snapshot, upgrade the etcd service with:
 
     juju upgrade-charm etcd
 
-This will handle upgrades between minor versions of etcd. Major upgrades from
-etcd 2.x to 3.x are currently unsupported. Instead, data will be run in etcdv2 stores over the etcdv3 api.
+This will handle upgrades between minor versions of etcd. Instructions on how to upgrade from 2.x to 3.x can be found [here](https://github.com/juju-solutions/bundle-canonical-kubernetes/wiki/Etcd-2.3-to-3.x-upgrade) in the juju-solutions wiki.
 
 ### Upgrade Kubernetes
 
 The Kubernetes Charms use snap channels to drive payloads.
 The channels are defined by `X.Y/channel` where `X.Y` is the `major.minor` release
-of Kubernetes (e.g. 1.6) and `channel` is one of the four following channels:
+of Kubernetes (e.g. 1.9) and `channel` is one of the four following channels:
```

Comment:
could you replace `e.g.` with `for example` - we're trying to remove the latin abbreviations in our documentation (per https://kubernetes.io/docs/home/contribute/style-guide/#avoid-latin-phrases)